modules/geolocation/geolocation.py

```diff
@@ -411,6 +413,17 @@ def __get_average_otherwise_median(self,trimmedArray,median):
             return np.average(trimmedArray,axis=0)
 
 
+    # Helper function required for input
+    def concatenate_locations(self, newLocations):
+
+        self.__locationsList = self.__locationsList + newLocations
+
+
+    def convert_NP_from_list(self, locationsList):
```

Comment:
Does this really need to be a wrapper function rather than just a direct call to np.array(...)? Only seems to be used once right now.